to.etc.domui/src/main/java/to/etc/domui/dom/html/NodeBase.java

```diff
@@ -188,6 +188,8 @@
 	 */
 	static private final byte F_NOREPLACE = 0x08;
 
+	static private final byte F_RENDERSTYLES = 0x16;
```

Comment:
Should be called F_STYLERENDERED, because it is telling you not to RENDER styles but _that_ a style was rendered last time around.